public/scss/components/_offramp.scss

```diff
@@ -0,0 +1,18 @@
+.offramp {
+    width: 90%;
+
+    p {
+        margin-bottom: $space-lg;
+    }
+
+    div {
+      p {
+          margin-bottom: $space-md;
+      }
+
+      ul li {
+          margin-bottom: $space-xxs;
+        }
+      
+    } 
+}
```

Comment:
In general, we should have general `p` tag rules that work across the whole app than just for these pages. 

I have a PR in the works that changes some of our app-wide text formatting. Let's take this out for now.

Reply:
Save it for later though and we can see if it's needed.